tests/src/lpp_tests.rs

```diff
@@ -162,7 +168,7 @@ fn open_loan_unauthorized_contract_id() {
         .execute(
             test_case.address_book.lpp().clone(),
             test_case.address_book.lpp().clone(),
-            &lpp::msg::ExecuteMsg::OpenLoan {
+            &lpp::msg::ExecuteMsg::<Lpns>::OpenLoan {
```

Comment:
[everywhere in the package]
1. why do we use the type as fully qualified?
2. why do we copy/paste `ExecuteMsg::<Lpns>` and `QueryMsg::<Lpns>` in many places?

 please fix